swarm/pss/notify/notify.go

```diff
@@ -0,0 +1,387 @@
+package notify
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
```

Comment:
@nolash if you are using `Warn` and `Error` log calls, probably best to use `swarm/log` package, so that we actually count and review those when people use PSS from our cluster.